dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/job/TrackerNotificationThread.java

```diff
@@ -76,6 +78,8 @@ public void call()
             {
                 serviceMapper.get( object.getClass() ).accept( object.getId() );
             }
+
+            log.info( "Tracker notification side effects completed" );
```

Comment:
Should this really be log level info? I think maybe we should use the async notifier system for this? like we do for other info messages from the tracker (since its async now)